db/seeds.rb

```diff
@@ -5,3 +5,18 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+User.create!([
+  { email: "jonarnett90@gmail.com",       password: "password", full_name: "Jonathan Arnett" },
```

Comment:
Let's put together some ipsum names and ```@example.com``` email addresses, rather than expose work and personal addresses in a public repo.